.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/dmlc/dmlc-core.git
 [submodule "3rdparty/ps-lite"]
 	path = 3rdparty/ps-lite
-	url = https://github.com/dmlc/ps-lite
+	url = https://github.com/threeleafzerg/ps-lite.git
```

Comment:
Please remember to revert after https://github.com/dmlc/ps-lite/pull/137 is merged.

Reply:
Certainly.